lib/models/device.coffee

```diff
@@ -509,6 +544,76 @@ exports.move = (uuid, application, callback) ->
 	.nodeify(callback)
 
 ###*
+# @summary Start application on device
+# @name start
+# @public
+# @function
+# @memberof resin.models.device
+#
+# @param {String} uuid - device uuid
+# @fulfil {String} - application container id
+# @returns {Promise}
+#
+# @example
+# resin.models.device.start('7cf02a6').then(function(containerId) {
+# 	console.log(containerId);
+# });
+#
+# @example
+# resin.models.device.start('7cf02a6', function(error, containerId) {
+# 	if (error) throw error;
+# 	console.log(containerId);
+# });
+###
+exports.start = (uuid, callback) ->
```

Comment:
@Page- Sounds like a good plan.

@lekkas What Page means is that you can rename `.start()` to `.startApplication()`, `.stop()` to `.stopApplication()` and edit the definition of `.restart()` to be `exports.restart = exports.restartApplication = ( ... ) ->`, so we maintain backwards compatibility.

My only worry is how this will play with the generated documentation. If you have a way to display both `.restar()` and `.restartApplication()` in `DOCUMENTATION.md` in a way that they share the function details, but the former is marked a "deprecated", we should do it.